gradle.properties

```diff
@@ -3,6 +3,6 @@
 org.gradle.jvmargs=-Xmx4G
 org.gradle.daemon=false
 minecraft_version=1.18.2
-forge_version=40.1.80
+forge_version=40.2.17
```

Comment:
Bumped the version as high as I could while debugging. Is there any reason _not_ to be on the latest version of Forge compatible with 1.18.2?

Reply:
Usually it's better to stick with the recommended versions rather than latest due to stability, but in general this shouldn't be problematic for us.

Reply:
Am happy to drop it to 40.2.0 if you want, or try bumping up to 40.2.18 again (the highest version of Forge that's 1.18.2 compatible and likely the final version IIUC), though I had issues with gradle accessing all the files it needed when I tried 40.2.18 before (hence using second latest). 

If I've time to test later, might do a check to make sure builds made using the latest API can still work on earlier versions of the same minor, since my own setup I've been devving against is to make everything use the latest version of everything as much as is possible. Should probably also check what version TFC 1.18 is compiled against and make sure a modpack running in that version of Forge can still use Blaze Map without issue (it theoretically should, as I didn't encounter any notable API changes).

Reply:
I don't think there's a reason for this not to work as far back as Forge 40.0.0, our contact surface is very small.